phpstan.neon

```diff
@@ -1,13 +1,6 @@
-includes:
-    - vendor/phpstan/phpstan-doctrine/extension.neon
-    - vendor/phpstan/phpstan-symfony/extension.neon
-    - vendor/phpstan/phpstan-webmozart-assert/extension.neon
-
 parameters:
     reportUnmatchedIgnoredErrors: false
```

Comment:
would remove this. as far as i see, we only have a single ignore.

Reply:
this is copied from sylius plugin skeleton would keep it